cmd/node/flags.go

```diff
@@ -322,6 +322,12 @@ var (
 			"and by advanced users, as a too high memory ballast could lead to Out Of Memory panics. The memory ballast " +
 			"should not be higher than 20-25% of the machine's available RAM",
 	}
+
+	// generateKey defines a flag that, if set, will generate every time when node starts a new signing key
+	generateKey = cli.BoolFlag{
+		Name:  "generate-key",
+		Usage: "Boolean flag for enabling the node to generate a signing key when it starts (if the validatorKey.pem file is present, this flag is not required)",
```

Comment:
optional: if the validatorKey.pem file is present, setting this flag to true will overwrite the BLS key used by the node

Reply:
done